dotnet/Capstone/DAO/BirdSqlDao.cs

```diff
@@ -199,11 +265,12 @@ public List<Bird> listAllBirds()
         private Bird MapRowToBird(SqlDataReader reader)
         {
             Bird bird = new Bird();
-            bird.id = Convert.ToInt32(reader["id"]);
-            bird.name = Convert.ToString(reader["name"]);
-            bird.description = Convert.ToString(reader["description"]);
-            bird.imgUrl = Convert.ToString(reader["picture"]);
-         
+            bird.BirdId = Convert.ToInt32(reader["id"]); 
+            bird.ListId = Convert.ToInt32(reader["list_id"]);
+            bird.BirdName = Convert.ToString(reader["name"]);
+            bird.ImgUrl = Convert.ToString(reader["picture"]);
+            bird.ZipCode = Convert.ToString(reader["zip_code"]);
```

Comment:
Because these items are not marked "NOT NULL" in the schema, there's a chance for them to cause errors when pulling a null value (I believe SQL Server null values aren't innately converted to C# null values). See the [GetSafeString](https://github.com/ahdisease/Bird-Finder/blob/654e710664ed8688628956d2bd7b0a53f0878d21/dotnet/Capstone/DAO/UserProfileSqlDao.cs#L238) method in UserProfileSqlDao for an example of how to handle this.